src/components/NcAppNavigationSearch/NcAppNavigationSearch.vue

```diff
@@ -0,0 +1,252 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<docs>
+### Usage
+
+```vue
+<template>
+	<div class="styleguide-wrapper">
+		<NcContent app-name="styleguide-app-navigation" class="content-styleguidist">
+			<NcAppNavigation>
+				<template #search>
+					<NcAppNavigationSearch :value.sync="searchQuery">
+						<template #actions>
+							<NcActions aria-label="Filters">
+								<template #icon>
+									<IconFilter :size="20" />
+								</template>
+								<NcActionButton>
+									<template #icon>
+										<IconAccount :size="20" />
+									</template>
+									Filter by name
+								</NcActionButton>
+								<NcActionButton>
+									<template #icon>
+										<IconCalendarAccount :size="20" />
+									</template>
+									Filter by year
+								</NcActionButton>
+							</NcActions>
+							<NcButton aria-label="Search globally" type="tertiary">
+								<template #icon>
+									<IconSearchGlobal :size="20" />
+								</template>
+							</NcButton>
+						</template>
+					</NcAppNavigationSearch>
+				<template #list>
+					<NcAppNavigationItem name="First navigation entry">
+						<template #icon>
+							<IconStar :size="20" />
+						</template>
+					</NcAppNavigationItem>
+					<NcAppNavigationItem name="Second navigation entry">
+						<template #icon>
+							<IconStar :size="20" />
+						</template>
+					</NcAppNavigationItem>
+				</template>
```

Comment:
Is it supposed to be without the list in the docs?

![image](https://github.com/user-attachments/assets/cd2be4e8-9d45-47e8-9ce6-2e7f871e7e6b)

Reply:
I thought having the list there would make it more clearer?

Reply:
I mean, there is the list in the example's code, but not rendered...

Reply:
See new comment

Reply:
Already fixed :sweat_smile: